docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -401,6 +415,489 @@ docker compose up
+
+:::info
+
+In the future, the Anchor Platform will also support making payments associated with deposit transactions to users on behalf of the business. This is another reason why you may want to use the Anchor Platform's event system.
+
+:::
+
+:::caution
+
+User account may not be ready to receive funds. You should check that account has established the [trustline](/docs/glossary#trustline). Otherwise, set transaction to status `pending_trust` to indicate that anchor is waiting for the user establishing the trustline.
+
+:::
+
+:::tip
+
+(Optional) You may set transaction status to `pending_stellar`, when it has been submitted to the network, but not yet included in the ledger. Due to the nature of Stellar network, transaction confirmation is almost instant. Some SDKs (Java) have blocking transaction submission method. After receiving success result from the submission method, transaction will already be completed.
+
+:::
+
+#### Completing Transaction
+
+Finally, after transaction has been sent and successfully submitted to the network, it's time to complete transaction:
+
+<CodeExample>
+
+```bash
+transaction='
+{
+  "id": '"$transaction_id"',
+  "status": "completed",
+  "message": "completed",
+  "stellar_transactions": [
+    {
+      "id": "<stellar_transaction_id>",
+      "payments": [
+        {
+          "id": "<payment_operation_id>",
+          "amount": {
+            "amount": "10",
+            "asset": "USDC:GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5"
+          }
+        }
+      ]
+    }
+  ]
+}
+'
+
+curl localhost:8080/transactions \
+    -X PATCH \
+    -H 'Content-Type: application/json' \
+    -d '{"records":['"$transaction"']}'
+```
+
+</CodeExample>
+
+:::caution
+
+It's possible to send multiple transactions to fulfill user request, but due to current protocol restrictions, only the latest one will be visible for user via `stellar_transaction_id` field.
+
+:::
+
+#### Handling Errors
+
+When unrecoverable error happens, it's required to set transaction status to `error`. Message field can be used to describe error details.
+
+<CodeExample>
+
+```bash
+transaction='
+{
+  "id": "<transaction_d>",
+  "status": "error",
+  "message": "Example error"
+}
+'
+
+curl localhost:8080/transactions \
+    -X PATCH \
+    -H 'Content-Type: application/json' \
+    -d '{"records":['"$transaction"']}'
+```
+
+</CodeExample>
+
+:::tip
+
+This status should not be used after user has made a transfer (if there are some external factors that prevent completing transaction). Instead, refund should be initiated. It can be done either by user's request (in your web view), or automatically.
+
+:::
```

Comment:
Lets add an example for updating the platform of a refund payment.

Reply:
TBD in another PR